src/store/socket.js

```diff
@@ -288,14 +305,24 @@ class LiveSession {
     const {
       gamestate,
       isLightweight,
+      isRevealGrimoire,
       isNight,
       nomination,
       votingSpeed,
       votes,
       lockedVote,
       isVoteInProgress,
-      fabled
+      fabled,
+      bluffs
     } = data;
+    if (isRevealGrimoire) {
+      // includes ALL roles, reminders & bluffs
+      const popup =
+        "Reveal the Storyteller's grimoire? This will overwrite your grimoire.";
+      if (!confirm(popup)) {
```

Comment:
This is not going to work in many browser, because opening a `confirm` without user interaction is very often unwanted behaviour...

Reply:
Something like this?
![Untitled](https://user-images.githubusercontent.com/14160941/117563097-8011c600-b09b-11eb-87d6-11018865ac10.png)

Reply:
No, please no second grimoire! Yes, showing a button in the center when there's a "revealed" dataset from the ST available to show would probably be best.